language-support/ts/codegen/tests/ts/generated/src/__tests__/test.ts

```diff
@@ -5,6 +5,7 @@ import { ChildProcess, spawn } from 'child_process';
 import waitOn from 'wait-on';
 import { encode } from 'jwt-simple';
 import Ledger from '@digitalasset/daml-ledger-fetch'
+import { CreateEvent, ArchiveEvent } from  '@digitalasset/daml-ledger-fetch'
```

Comment:
```suggestion
import Ledger, { CreateEvent, ArchiveEvent } from  '@digitalasset/daml-ledger-fetch'
```
and remove the previous line.